File: middleware/event/event_type.py
from enum import Enum

class EventType(Enum):
    SUBSCRIBE_TOPIC_REQUESTED = "subscribe_topic_requested"
    PUBLISH_MESSAGE_REQUESTED = "publish_message_requested"
    MESSAGE_ARRIVED = "message_arrived"
    CONTEXT_UPDATED = "context_updated"
    ACTION_TAKEN = "action_taken"
    TASK_NOTIFIED = "task_notified"  # Deprecated?
    TASK_INITIATED = "task_initiated"
    TASK_TERMINATED = "task_terminated"
    USER_NOTIFIED = "user_notified"

File: lapras-middleware/src/main/java/kr/ac/kaist/cdsn/lapras/event/EventType.java
package kr.ac.kaist.cdsn.lapras.event;

/**
 * Created by Daekeun Lee on 2016-11-11.
 */
public enum EventType {
    SUBSCRIBE_TOPIC_REQUESTED,
    PUBLISH_MESSAGE_REQUESTED,
    MESSAGE_ARRIVED,
    CONTEXT_UPDATED,
    ACTION_TAKEN,
    TASK_NOTIFIED, /*Deprecated?*/
    TASK_INITIATED,
    TASK_TERMINATED,
    USER_NOTIFIED
}
